shop-be/product-service/serverless.ts

```diff
@@ -18,11 +21,16 @@ const serverlessConfiguration: AWS = {
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
       NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+      POSTGRESQL_HOST: process.env.POSTGRESQL_HOST,
+      POSTGRESQL_PORT: '5432',
```

Comment:
Also it can be easier just to move to the secrets the whole connection string, not piece by piece

Reply:
done